src/jemalloc_cpp.cpp

```diff
@@ -39,6 +39,20 @@ void	operator delete(void *ptr, std::size_t size) noexcept;
 void	operator delete[](void *ptr, std::size_t size) noexcept;
 #endif
 
+#if __cpp_aligned_new >= 201606
+/* C++17's over-aligned operators. */
+void    *operator new(std::size_t size, std::align_val_t);
```

Comment:
Nit: no spaces necessary here. (Or if you have a zeal for extra whitespace, it should be a tab).